src/constants/platform/url.ts

```diff
@@ -22,9 +22,9 @@ export const GeyserPodcastUrl = 'https://fountain.fm/show/P5kWpoVIT7YGmPRenVN4'
 export const GeyserGithubUrl = 'https://github.com/geyserfund/geyser-app'
 export const GeyserHomepageUrl = 'https://geyser.fund/'
 export const GeyserPrivacyUrl =
-  'https://geyser.notion.site/Geyser-Privacy-Policy-7d008651271249b18d6886edbb2edaba'
+  '/privacy-policy'
 export const GeyserTermsAndConditionsURL =
-  'https://geyser.notion.site/Terms-Conditions-815ff42c97af4cdf8861011a8a4c1fe0'
+  '/terms-and-conditions'
 
```

Comment:
Since we are moving away from the external link to an internal link. For all internal links, we maintain and update the: routerPaths.ts which has PathName and getPath functions.
I suggest we move these to over there as well, for consistency.